docs/explore/introduction/contributing.md

```diff
@@ -14,80 +14,145 @@ tags:
 
-API3 is open to any proposals and improvements. Please feel free to contribute.
+We think critical infrastructure, like API3 DAO's first-party oracles, should
+implement open and transparent contributor processes to align to our vision of
+being a programmatically driven protocol.
+
+To support DAO governance and inform voters, all teams publish a public proposal
+at regular intervals, meaning strategy & operations are accessible on the
+[API3 Governance Forum](https://forum.api3.org/) for all to understand.
+Additionally, teams generally update the community with retrospective reports.
+
+## Types of contributors
+
+Community participants within a Decentralized Autonomous Organization (DAO) have
+an active role in shaping the future of the project. Our aim is to simplify the
+process for those who wish to participate, providing predefined methods for
+engagement. Generally these take forum in three routes:
+
+- Core team contributions 
+- Completion of pre-defined bounties 
+- New initiatives or ideas
+
+### Contributors can engage with proposals made by the core teams
+
+If contributors feel they possess particular skills or expertise that may
+benefit the proposal's execution, they can directly communicate with the
+proposal leads. This collaborative approach encourages a diverse array of
+viewpoints and fosters a proactive culture where everyone's skills and
+experiences are leveraged for the benefit of the DAO's objectives.
+
+### Take on pre-defined bounties
+
+Community participants have the opportunity to engage with pre-set bounties,
+organized in a modular structure to offer clear and attainable tasks. For
+instance, a bounty might be created for a specific job that is expected to be
+completed within two days. This structure allows any member of the community to
+take on the job, complete it, and subsequently receive the designated grant as a
+reward retrospectively.
+
+### Submit your own proposal
+
+We welcome proposals that bring new ideas or sub-teams to the DAO. Generally
+engaging the DAO through our community in a constructive manner will help
+sentiment check a proposal and help shape it to align to current contributors.
+If you are proposing an initiative that would require collaboration with a
+current sub team, we suggest co-ordinating proactively and outlining how you
+will work together.
+
+# Put forward your ideas to the API3 DAO
+
+Regardless of the contributor's skill, API3's vision is for participation to be
+as frictionless as possible. Individuals with a relevant background can
+contribute to the project in a flexible manner, replicating similar
+freelancer/consultancy type figures commonly seen.
+
+## Technical contributors
+
+For technical folks this could be through development, integrations, writing
+documentation, etc. Although the general challenge is that one needs to be able
+to use a solution proficiently before being able to improve it. If this applies
+we suggest digesting the get started materials.
```

Comment:
A link would be helpful for "getting started"

Reply:
updated